content/en/docs/v3.5/integrations.md

```diff
@@ -103,6 +103,10 @@ The sections below list etcd client libraries by language.
 - [marshall-lee/etcd.erl](https://github.com/marshall-lee/etcd.erl) - Supports v2
 - [zhongwencool/eetcd](https://github.com/zhongwencool/eetcd) - Supports v3+ (GRPC only)
 
+### Elixir
+
+- [team-telnyx/etcdex](https://github.com/team-telnyx/etcdex) - Supports v3+ (GRPC only)
```

Comment:
@ptabor might have a comment. 

https://github.com/etcd-io/bbolt/issues/413#issuecomment-1452062106

Reply:
Hey @ahrt, @ptabor - Keen to get a decision on this pr so we can either close or merge. Do you have any objection to this project being included in our list? I can see it is being actively maintained, has a few stars, more than 1 contributor and is in a language that we don't already have an implementation for so my preference would be to include it.

Reply:
Hey @ahrtr, @spzala  - Can you please take another look at this one?  Keen to either close or merge rather than leave sitting stale. We have given plenty of time for lazy consensus in relation to above comment.

Reply:
Not willing to sell or anything, but this is the only pure Elixir implementation of an etcd client you'll find.

Reply:
Thanks @balena . Are there any applications which are using [team-telnyx/etcdex](https://github.com/team-telnyx/etcdex) to communicate with etcd cluster?

Reply:
That I know of, private sources are using it.